docker/Makefile

```diff
@@ -35,7 +35,8 @@ unit-tests-ci:
 .PHONY: build
 build:
 	npm install
-	docker-compose build mail app ld-api next-proxy next-app
+    # TODO: When this Makefile is moved to repo root, turn "../" below into "./"
+	LFMERGE_VERSION="$(shell jq -r .lfmerge ../other-dependencies.json)" docker-compose build mail app ld-api next-proxy next-app
```

Comment:
did you happen to check whether doing this type of thing in the `docker-compose.yml` is possible?  I'm thinking something like `ghcr.io/sillsdev/lfmerge:${jq ...}` or 
```yml
environment:
    - LFMERGE_VERSION=`jq ...`
```

I'm not certain off the top of my head but would be better if possible.

Reply:
I checked, and you can't use shell commands inside the docker-compose.yml file. I agree that it would have been simpler — it's the first thing that I tried — but no, if you want things that will change you have to pass them in as environment variables. (Or pass the docker-compose file through sed or something, but the documentation recommends env vars).

Reply:
I proposed a different idea that would eliminate some of these work-arounds.